src/cardHolder.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import './cardHolder.css';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
```

Comment:
You can do `import { Card, CardActionArea, ..., ... } from @material-ui/core` for simplicity

Reply:
Also it's better to separate files related to smaller components in `./src/components`

Reply:
i tried doing that, but idk why it keeps running into an error saying it cannot find those files? i feel like I'm missing something ?!